### PR TITLE
UML-579 : content changes on viewer UI and feature tests amended

### PR DESCRIPTION
**Pull request by SeemaMenon**
## Purpose
https://opgtransform.atlassian.net/browse/UML-579

Fixes UML-579
Altered content and the feature test content for it to run successfully

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Refunds service_

## Checklist

* [X] I have performed a self-review of my own code
* [NA] I have added relevant logging with appropriate levels to my code
* [NA] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig [hunk 1]
```diff
@@ -11,12 +11,12 @@
             <div class="govuk-grid-row">
```
Comment (GemTay) [inline service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig:11]: Change the tab heading too on line 3

Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig:11]: Done... 👍

File: service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
 
-                    <h1 class="govuk-heading-xl">Cancelled code</h1>
+                    <h1 class="govuk-heading-m">The access code you entered has been cancelled</h1>
 
-                    <p class="govuk-body">The code you entered has been cancelled.</p>
+                    <p class="govuk-body">To see this LPA, ask the donor or attorney you’re dealing with to make a new code.</p>
 
-                    <a href="{{ path('enter-code') }}" role="button" draggable="false" class="govuk-button govuk-button--start">
-                        Try again
+                    <a href="{{ path('enter-code') }}" class="govuk-link">
```
Comment (hawx) [inline service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig:18]: why did you change this from a button?

Comment (hawx) [reply service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig:18]: and if this is the new style, should the "code not found" be updated to match?

image

Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/check-code-cancelled.html.twig:18]: Is a good question @hawx . 
- Button changed to link as it's a requirement in the ticket.
- Nothing about 'Code not found page' was mentioned in the ticket though !

The ticket only said below requirements:
1. Viewer sees the reason the LPA can’t be found (it’s cancelled or expired) using the content above, on the pages shown on screenshots [in the ticket]
2. ‘Enter another code’ should be a link which take user back to the enter code page (which will be blank)

File: service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig [hunk 3]
```diff
@@ -11,12 +11,13 @@
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
 
-                    <h1 class="govuk-heading-xl">Expired code</h1>
+                    <h2 class="govuk-heading-xl">The access code you entered has expired</h2>
```
Comment (hawx) [inline service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig:14]: Why did you change this to a h2?

Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig]: mistake..that was! corrected. Thanks Josh

File: service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig [hunk 4]
```diff
@@ -11,12 +11,13 @@
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
 
-                    <h1 class="govuk-heading-xl">Expired code</h1>
+                    <h1class="govuk-heading-xl">The access code you entered has expired</h1>
```
Comment (mattmachell) [inline service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig:14]: `h1class` with no space is going to muck up your HTML parsing.

Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/check-code-expired.html.twig]: definitely yes! corrected.
